payjoin-ffi/src/receive/mod.rs

```diff
@@ -266,16 +265,23 @@ impl UninitializedReceiver {
         ohttp_keys: Arc<OhttpKeys>,
         expire_after: Option<u64>,
         amount: Option<u64>,
-    ) -> Result<InitialReceiveTransition, IntoUrlError> {
+        max_fee_rate_sat_per_vb: Option<u64>,
+    ) -> Result<InitialReceiveTransition, ImplementationError> {
         payjoin::receive::v2::Receiver::create_session(
             (*address).clone().into(),
-            directory,
+            directory.clone(),
```

Comment:
This clone was added because of the map_error below. Once that is revert it should be removed
```suggestion
            directory,
```